lib/network/modules/components/DirectionStrategy.js

```diff
@@ -3,7 +3,8 @@
  *
  * Strategy pattern for usage of direction methods for hierarchical layouts.
  */
-import TimSort from 'timsort';
+
+import { sort as timsort } from 'timsort';
```

Comment:
This isn't really obvious but Timsort has no default export. It works in Rollup since it's UMD. It doesn't however work in Mocha. This way it works in both build and test.